src/Profile/Profile.js

```diff
@@ -34,19 +37,30 @@ const useStyles = makeStyles((theme) => ({
   expandOpen: {
     transform: 'rotate(180deg)',
   },
+  cardText: {
+    align: 'left',
+    variant: "h6",
+    color: "textSecondary"
+  },
   avatar: {
-    backgroundColor: blue[500],
+    backgroundColor: '#00acc1',
   },
 }));
 
-export default function ResidentCard({resident}) {
+function ResidentCard({resident, exitProfileView}) {
   const classes = useStyles();
   const [expanded, setExpanded] = React.useState(false);
+  // const [close, setClose] = React.useState(false)
 
   const handleExpandClick = () => {
     setExpanded(!expanded);
   };
 
+  const handleCloseClick = () => {
```

Comment:
This is way more adept than just refreshing the page.

Reply:
I'd like to review this when we have a moment.